typespec-extension/src/code-model-builder.ts

```diff
@@ -1980,29 +1980,18 @@ export class CodeModelBuilder {
         : this.doubleSchema;
 
     const choices: ChoiceValue[] = [];
-    const getUnionVariant = (member: any) => {
-      if (member.variant) {
-        return member.variant as UnionVariant;
-      } else if (member.type) {
-        // TODO: remove this; in current runtime, the UnionVariant is from this "type" instead of "variant"
-        return member.type as UnionVariant;
```

Comment:
typespec-azure-core has a break from 0.39.0 to 0.39.1 on this.

Hence the patch version bump in dependencies.